### DISCUSSION TITLE
카카오맵에서의 동작 여부 (네이버맵 비교)

### DISCUSSION THREAD

**Discussion by J-Kim-Korea** [changdoc/naver-vaccine-macro#196] [Q&A]
좋은 스크립트 만들어 주셔서 감사합니다.
네이버맵에서 확인하는게 2초 리프레쉬레이트가 있어, 발견 후에도 예약 실패가 잦은 것 같아.
카카오맵에서 확인하는 방식으로 해보니, 0.5초 정도로 훨씬 짧게 수량 확인이 가능한 것 같아 좋은 것 같습니다.

해주신 이전 설명들을 찾아보니, 카카오맵은 화면에 마커의 수량은 변하지 않지만, left_count 데이터를 보고 네이버로 연결시켜주는 것으로 이해했는데요.
기존 네이버 맵 방식과, 카카오맵 방식을 둘다 띄워서 해보는 중인데,
네이버 맵에서는 수량 파악이 되어, 예약 페이지로 넘어가면서 잘 작동하는 것 같은데,
카카오맵에는 깜깜 무소식이네요..

설명해 주신데로 셋팅은 마쳤고, left_count 데이터 계속 에러없이 잘 들어오는데, 
원인이 무엇인지 잘 모르겠습니다ㅠㅠ

혹시 짐작가시는 이유가 있는지, 알려주시면 감사하겠습니다.

**Comment by changdoc** [comment]
우선 파악하신대로 동작중이라 Left count가 0이상이 탐지됐다면 네이버 예약 신청 페이지로 이동하는 로직이 완전 동일한 상황이라

카카오맵에서 추출한 request 구역과 네이버 맵 graphq로 생성한 병원 목록간의 차이가 있는건 아닌지 확인이 필요할 것 같습니다.

카카오 측에서 매크로로 요청하는 건에 대해서만 데이터가 안내려오게 막힌건지... 파악이 안되긴했는데 그경우도 생각해야할거 같긴하네요.
